kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/ConvertUtils.java

```diff
@@ -171,11 +173,26 @@ public static QueryFilter buildQueryFilter(Integer page, Integer pageSize) {
     }
 
     public static QueryFilter buildQueryFilter(Integer page, Integer pageSize, String orderBy, boolean asc) {
+        return buildQueryFilter(page, pageSize, orderBy, asc, null);
+    }
+
+    public static QueryFilter buildQueryFilter(Integer page, Integer pageSize, String orderBy, boolean asc, String filter) {
```

Comment:
I think that "general" query builder shouldn't contain specific implementation like this task name filtering. I think that this method should be renamed to something like buildTaskQueryFilter, retrieve query filter from general method and apply task specific stuff here.